File: others/yksdyWeb/libs/yahoo web service-2.11/yws-2.11/PHP/simpleSearch.php
<?php
// hello.php
// Requirements:
// allow_url_open must be set to true (for file_get_contents)
//

// The Yahoo! web services request
$request = 'http://search.yahooapis.com/WebSearchService/V1/webSearch?appid=YahooDemo&query=madonna&results=2';

// Make the request
$results = file_get_contents($request);

if ($results == false) {
    die("Web services request failed");
}

// Output the XML
echo htmlspecialchars($results, ENT_QUOTES);
?>

File: others/yksdyWeb/libs/yahoo web service-2.11/yws-2.11/Java/simpleSearch.java
/**
 * Simple Search using Yahoo! Web Services
 *
 * @author Daniel Jones
 * Copyright 2006 Daniel Jones
 * Licensed under BSD open source license
 * http://www.opensource.org/licenses/bsd-license.php
*/

import java.io.InputStream;
import java.net.URL;

public class YahooWebServiceExample {

       public static void main(String[] args) {
               String request = "http://search.yahooapis.com/WebSearchService/V1/webSearch?appid=YahooDemo&query=madonna&results=2";
               try {
                       URL url = new URL(request);
                       InputStream in = url.openStream();
                       byte[] buf = new byte[1024];
                       int len;
                       while ((len = in.read(buf)) > 0) {
                               for (int i = 0; i < len; i++) {
                                       System.out.print((char) buf[i]);
                               }
                       }
                       in.close();
               } catch (Exception e) {
                       System.out.println("Web services request failed");
               }
       }
}
